sublime/User/Package Control.sublime-settings

```diff
@@ -5,39 +5,47 @@
 	],
 	"installed_packages":
 	[
+		"A File Icon",
 		"All Autocomplete",
 		"Better CoffeeScript",
-		"Color Highlighter",
-		"DashDoc",
-		"Delete Current File",
 		"Diffy",
-		"EncodingHelper",
-		"File Rename",
+		"Ecmascript Syntax",
+		"Edge Theme",
+		"Elixir",
+		"ElixirSyntax",
+		"FileManager",
 		"Git",
-		"GitGutter",
 		"Jade",
-		"Markdown Preview",
-		"Materialize",
+		"Javascript Beautify",
+		"JavaScriptNext - ES6 Syntax",
+		"Laravel Blade Highlighter",
```

Comment:
Would it be affordable to exclude this from the PR? Or it will be easier to iterate with a new commit?

Asking about it 'cos we don't use laravel nor blade in weLaika ;)